src/data/wallets/wallet-data.ts

```diff
@@ -408,6 +409,76 @@ export const walletsData: WalletData[] = [
     onboard_documentation: "https://docs.safe.global/",
     documentation: "https://docs.safe.global/",
   },
+  {
+    last_updated: "May 10, 2024",
+    name: "Coin wallet",
```

Comment:
Per usage on https://coin.space/

```suggestion
    name: "Coin Wallet",
```